exe/sufia_export

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+require 'rails'
+
+# get the current running directory & load the environment from there
+require ::File.join(Dir.pwd, 'config/environment')
+
+options = {}
+
+args = OptionParser.new do |opts|
+  opts.banner = "Usage: sufia_export [options]"
+
+  opts.on("-v", "--[no-]verbose", "Run verbosely") do |v|
+    options[:verbose] = v
+  end
+
+  opts.on("--limit NUMBER_OF_OBJECTS", Integer, "Export a maximum number of objects (defaults to all)") do |v|
+    options[:limit] = v
+  end
+
+  opts.on("--ids ID_LIST", Array, "Expoprt only the objects in the list of IDs") do |v|
+    options[:ids] = v
+  end
+
+  opts.on("--models MODEL_LIST", Array, "Specify the models to be output (defaults to [Collection,GenericFile]",
+          "To override the default converter for a model or to register a new converter add the converter after the model.",
+          "For example: --models \"GenericFile Mine::NewGenericFileConverter\"",
```

Comment:
So if I want to pass in two models with new converters, would I invoke `sufia_export` like this?

```bash
$ sufia_export ... --models [Collection Mine::NewCollectionConverter,GenericFile Mine::NewGenericFileConverter]
```

Am unclear on whether `MODEL_LIST` is supposed to look like a Ruby array or a string or what.